docker-compose.base.yml

```diff
@@ -158,6 +158,7 @@ services:
             PGUSER: posthog
             PGPASSWORD: posthog
             DEPLOYMENT: hobby
+            CDP_API_URL: 'http://plugins:6738'
```

Comment:
just to check this is fine in the base and not in the hobby yml

Reply:
I don't see why not. If you are running the django stuff in docker in any situation it will need to talk to the relevant plugins docker service